### PR TITLE
Change Internet to Ethernet in setting-up.adoc

### PR DESCRIPTION
**Pull request by JohnVeness**
Alternatively, could say "wired network connection". But in any case, a USB-to-Ethernet adapter is not necessarily connecting to *Internet*.

### CODE REVIEW

**Review by nathan-contino — APPROVED** [review]
LGTM

**Comment by aallan** [comment]
Don't like the repeated use of the word "Ethernet", if people hate "internet connection" we can change it to "network" perhaps, _"If your Raspberry Pi doesn't have an Ethernet port, you can still connect to a wired network using a USB-to-Ethernet adapter."_

File: documentation/asciidoc/computers/getting-started/setting-up.adoc [hunk 1]
```diff
@@ -146,6 +146,6 @@ The following Raspberry Pi models come with WiFi and Bluetooth connectivity:
 * Raspberry Pi Zero W
 * Rsapberry Pi Zero 2 W
 
-The "Model B" suffix indicates variants with an Ethernet port; "Model A" indicates no Ethernet port. If your Raspberry Pi doesn't have an Ethernet port, you can still connect to a wired internet connection using a USB-to-Ethernet adapter.
+The "Model B" suffix indicates variants with an Ethernet port; "Model A" indicates no Ethernet port. If your Raspberry Pi doesn't have an Ethernet port, you can still connect to a wired Ethernet connection using a USB-to-Ethernet adapter.
```
Comment (nathan-contino) [inline documentation/asciidoc/computers/getting-started/setting-up.adoc:149]:
```suggestion
The "Model B" suffix indicates variants with an Ethernet port; "Model A" indicates no Ethernet port. If your Raspberry Pi doesn't have an Ethernet port, you can still connect to a wired network using a USB-to-Ethernet adapter.
```

**Comment by nathan-contino** [comment]
Agreed, updated!

**Comment by LizUpton** [comment]
Closing: fiddly and expending too many eyeballs.

**Pull request closed without merging**